ci/dra_aarch64.sh

```diff
@@ -9,26 +9,49 @@ export JRUBY_OPTS="-J-Xmx1g"
 # Extract the version number from the version.yml file
 # e.g.: 8.6.0
 # The suffix part like alpha1 etc is managed by the optional VERSION_QUALIFIER_OPT environment variable
-STACK_VERSION=`cat versions.yml | sed -n 's/^logstash\:\s\([[:digit:]]*\.[[:digit:]]*\.[[:digit:]]*\)$/\1/p'`
+STACK_VERSION=`cat versions.yml | sed -n 's/^logstash\:[[:space:]]\([[:digit:]]*\.[[:digit:]]*\.[[:digit:]]*\)$/\1/p'`
 
-if [ -z "$VERSION_QUALIFIER_OPT" ]; then
-  RELEASE=1 rake artifact:docker
-  RELEASE=1 rake artifact:docker_oss
-  rake artifact:dockerfiles
+# WORKFLOW_TYPE is a CI externally configured environment variable where, if not an empty string,
+# it's assumed to be snapshot
+if [ -z "$WORKFLOW_TYPE"]; then
+  STACK_VERSION=${STACK_VERSION}-SNAPSHOT
+fi
+
+if [ -z "$WORKFLOW_TYPE"]; then
+  if [ -z "$VERSION_QUALIFIER_OPT" ]; then
+    RELEASE=1 rake artifact:docker
+    RELEASE=1 rake artifact:docker_oss
+    rake artifact:dockerfiles
+  else
+    VERSION_QUALIFIER="$VERSION_QUALIFIER_OPT" RELEASE=1 rake artifact:docker
+    VERSION_QUALIFIER="$VERSION_QUALIFIER_OPT" RELEASE=1 rake artifact:docker_oss
+    VERSION_QUALIFIER="$VERSION_QUALIFIER_OPT" rake artifact:dockerfiles
+    # Qualifier is passed from CI as optional field and specify the version postfix
+    # in case of alpha or beta releases:
+    # e.g: 8.0.0-alpha1
+    STACK_VERSION="${STACK_VERSION}-${VERSION_QUALIFIER_OPT}"
+  fi
 else
-  VERSION_QUALIFIER="$VERSION_QUALIFIER_OPT" RELEASE=1 rake artifact:docker
-  VERSION_QUALIFIER="$VERSION_QUALIFIER_OPT" RELEASE=1 rake artifact:docker_oss
-  VERSION_QUALIFIER="$VERSION_QUALIFIER_OPT" rake artifact:dockerfiles
-  # Qualifier is passed from CI as optional field and specify the version postfix
-  # in case of alpha or beta releases:
-  # e.g: 8.0.0-alpha1
-  STACK_VERSION="${STACK_VERSION}-${VERSION_QUALIFIER_OPT}"
+  # WORKFLOW_TYPE is set, it has value "SNAPSHOT"
+  if [ -z "$VERSION_QUALIFIER_OPT" ]; then
+    RELEASE=1 rake artifact:docker
```

Comment:
Do we need RELEASE=1 for snapshot?

Reply:
Yes you are right, I fooled myself with the ifs :-(